src/inverseRobot.js

```diff
@@ -4,10 +4,27 @@
  * @param {object} robot
  *
  * @return {object}
-*/
+ */
 
 function inverseRobot(robot) {
-  // write code here
+  const copy = {};
+  const arrKey = [];
+  const arrVal = [];
+
+  for (const key in robot) {
+    arrKey.unshift(key);
+    arrVal.unshift(robot[key]);
+  }
```

Comment:
I guess you can optimize it a bit by assigning arrKey to Object.keys(robot) and arrVal to Object.values(robot)
that way you wouldn't need to run this loop :)